net/haproxy/src/opnsense/mvc/app/models/OPNsense/HAProxy/HAProxy.xml

```diff
@@ -324,6 +324,26 @@
                     <Required>N</Required>
                 </customOptions>
             </stats>
+            <cache>
+                <enabled type="BooleanField">
+                    <default>0</default>
+                    <Required>N</Required>
+                </enabled>
+                <totalMaxSize type="IntegerField">
+                    <MinimumValue>1</MinimumValue>
+                    <MaximumValue>4095</MaximumValue>
```

Comment:
The MaximumValue does not reflect HAProxy's actual limits, if I understand the documentation correctly. I'd like to increase this value (post-merge) so that it fits more use-cases.